README.md

```diff
@@ -10,8 +10,10 @@ Both Social Team members and Node.js project collaborators submitting PRs should
 - There should generally be no hashtags contained within, unless they are relevant to real-world events like conferences.
   - Hashtags like #nodejs or #javascript are relatively unused outside of spam at this point, and add no additional semantic value on Twitter.
 - We do not amplify or share content that could unduly promote or benefit one corporation that does work in the space of Node.js over another.
-- Where possible, we try to amplify the collaborators who are doing the work rather than anonymously taking credit for it as the project.
-  - As an example, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
+- We try to amplify the collaborators who are doing the work rather than anonymously taking credit for it as the project.
+  - For general releases, we try to amplify those on the Release Team who are cutting releases directly rather than tweeting release announcements directly from the project's account.
+  - For security releases, it is important that the tweet be from an authoritative source that everyone will immediately recognize. So announcements for those releases are made directly from the project account.
```

Comment:
Smallest of nits, feel free to ignore:

```suggestion
  - For security releases, tweet directly from the project account because it is an authoritative source that readers will immediately recognize.
```